exercises/concept/chessboard/.docs/introduction.md

```diff
@@ -0,0 +1,94 @@
+# Introduction
+
+In Go, you can iterate a [slice](slices) using `for` but you could also use
+`range`. You will find this useful to iterate over a [map](maps).
```

Comment:
There are several links to outside resources. As per [the spec](https://github.com/exercism/docs/blob/main/anatomy/tracks/concept-exercises.md#file-docsintroductionmd):

> Links should be used sparingly, if at all. While a link explaining a complex topic like recursion might be useful, for most concepts the links will provide more information than needed so explaining things concisely inline should be the aim.

I'd thus suggest to remove the links.